src/core.js

```diff
@@ -1206,6 +1206,7 @@ define([
         }).on('remove-question', function (e) {
             if (!e.isInternal) {
                 var prev = _this.jstree("get_prev_dom", e.mug.ufid);
+                _this.showVisualValidation(null);
```

Comment:
You can simply omit the first argument like so: `_this.showVisualValidation();`

Will this clear all errors, including errors on questions that still exist? If yes, probably should make `showVisualValidation()` re-validate all mugs when no mug is provided.